scripts/pi-hole/php/header.php

```diff
@@ -266,23 +266,23 @@ function pidofFTL()
                             </li>
                             <!-- Menu Footer -->
                             <li class="user-footer">
-                                <!-- Update alerts -->
-                                <div>
+                                <!-- Version Infos -->
+                                <div class="<?php if(!isset($core_commit) && !isset($web_commit)) { ?>hidden-md <?php } ?>hidden-md hidden-lg">
```

Comment:
here too,
do the space after the by-php-added class into the php-code:
`<div class="<?php if(!isset($core_commit) && !isset($web_commit)) { ?>hidden-md <?php } ?>hidden-lg">`